siriuspy/siriuspy/timesys/csdev.py

```diff
@@ -605,15 +605,15 @@ def get_hl_trigger_database(hl_trigger, prefix=''):
     dbase['InInjTable-Mon'] = {
         'type': 'enum', 'enums': _et.ININJTAB, 'value': 0}
 
+    labs = '\n'.join(Const.HLTrigStatusLabels)
     dbase['StatusLabels-Cte'] = {
-        'type': 'char', 'count': 1000,
-        'value': '\n'.join(Const.HLTrigStatusLabels)
-        }
+        'type': 'char', 'count': len(labs) + 10, 'value': labs}
```

Comment:
Why are you adding 10 to them? Would it make sense to add a comment explaining it?